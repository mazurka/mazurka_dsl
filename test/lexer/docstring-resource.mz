#|
resource @testing is a collection of @users
|#
resource @testing