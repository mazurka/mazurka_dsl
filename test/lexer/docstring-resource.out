{docstring,1,<<"\nresource @testing is a collection of @users\n">>}.
{def_begin,4,testing,resource}.